packages/core/admin/admin/src/content-manager/components/DynamicTable/BulkActionsBar/index.js

```diff
@@ -3,8 +3,10 @@ import PropTypes from 'prop-types';
 import { Button } from '@strapi/design-system';
 import { useTracking } from '@strapi/helper-plugin';
 import { useIntl } from 'react-intl';
+import { useSelector } from 'react-redux';
 
 import ConfirmDialogDeleteAll from '../ConfirmDialogDeleteAll';
+import { listViewDomain } from '../../../pages/ListView/selectors';
```

Comment:
Could you remove the blank space between npm imports and local ones? I think we usually don't do this convention